certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Convert CSV data to JSON (deprecated)",
+        "description": "Convert CSV data to JSON"
+      }
+    },
+    "/ConvertCsvToJson": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertCsvToJson",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "dataIncludesHeader",
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "Data Includes Header?",
+            "description": "Data Includes Header?"
+          },
+          {
+            "in": "body",
+            "name": "csvData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CsvData"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "CSV",
+            "description": "CSV"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JsonResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Convert CSV to JSON",
+        "description": "Convert CSV to JSON"
+      }
+    },
+    "/ConvertJsonToExcel": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertJsonToExcel",
+        "consumes": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "AllInOneTable",
+            "type": "boolean",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "All in one table",
+            "description": "All in one table"
+          },
+          {
+            "in": "body",
+            "name": "JSON",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "JSON",
+            "description": "JSON"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileContentResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.